### ISSUE TITLE
Incorrect Loss Calculation

### ISSUE THREAD

**Issue by ddm7018** [kcinnick/pyredictit#14]
"If you sold all of your shares now, you would lose ${str(float(float(self.sell[:-1]) - float(self.avg_price[:-1])) * self.number_of_shares * 0.01 + (float(self.number_of_shares) * float(self.avg_price[:-1])) * 0.01)}"

Example:
Will @potus post 25 - 29 tweets from noon Feb. 21 to noon Feb. 28?
25.POTUSTWEETS.022817
You have 8 Yes shares.
Your shares have lost -$0.73 in value.
Your average purchase price of Yes shares is 67¢
Yes shares are currently being sold at 60¢
Yes shares are currently being bought for 58¢
If you sold all of your shares now, you would lose $4.640000000000001
The implied odds of this contract resolving to Yes are 60%
If this contract resolves to Yes, you would earn $1.48. Otherwise, you would lose $-5.84}

If you sold now you would get $4.64 with a 80 cents lost. Wouldn't that make more sense

**Comment by kcinnick** [comment]
yeah, I see what you mean.  I'll look into this later tonight - thanks!
